pagopa/pn-frontend
# Add mixpanel track profile properties and events

## Short description
This PR solve: PN-10175 - PN-10182 - PN-10193 - PN-10215
Track profile properties and added some new mixpanel events

## List of changes proposed in this pull request
- Add Profile Properties tracking ([PN-10175](https://pagopa.atlassian.net/browse/PN-10175))
- Add events on remove contacts success ([PN-10182](https://pagopa.atlassian.net/browse/PN-10182))
- Add IPD name on SEND_LOGIN_FAILURE ([PN-10193](https://pagopa.atlassian.net/browse/PN-10193))
- Add some properties to SEND_TOAST_ERROR and SEND_PAYMENT_DETAIL_ERROR ([PN-10215](https://pagopa.atlassian.net/browse/PN-10215))

## How to test
On PF and PF-login check on console if events described on the issues above

[PN-10175]: https://pagopa.atlassian.net/browse/PN-10175?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[PN-10182]: https://pagopa.atlassian.net/browse/PN-10182?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[PN-10215]: https://pagopa.atlassian.net/browse/PN-10215?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[PN-10193]: https://pagopa.atlassian.net/browse/PN-10193?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
